src/app/components/experiment-workspace-detail/experiment-workspace-detail.component.ts

```diff
@@ -98,8 +114,8 @@ export class ExperimentWorkspaceDetailComponent implements OnInit {
                 if (deletedObject.deleted.resourceKey?.uiTemplateTabId === this.tabId) {
                     this.deselectTab(null);
                 }
-                if (deletedObject.deleted.resourceKey?.uiTemplateTabId && deletedObject.deleted.resourceKey?.uiTemplateTabId in this.templateTabDescriptions) {
-                    delete this.templateTabDescriptions[deletedObject.deleted.resourceKey?.uiTemplateTabId];
+                if (deletedObject.deleted.resourceKey?.uiTemplateTabId && deletedObject.deleted.resourceKey?.uiTemplateTabId in this.templateTabObjects) {
```

Comment:
I don't think that the python like `in` comparison works in TypeScript...

Reply:
I checked it with the browsers debugger and it seems to work (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in).